### ISSUE TITLE
PODAAC ERS1 ERS2

### ISSUE THREAD

**Issue by JeremiahCurtis** [climate-mirror/datasets#289]
ftp://podaac-ftp.jpl.nasa.gov/allData/ers1/
ftp://podaac-ftp.jpl.nasa.gov/allData/ers2/

**Comment by erikfriesen** [comment]
Checking file sizes right now. Will update when complete.

**Comment by gofrogs2013** [comment]
The ers1 alone is nearly 30K files and 10K folders. I eventually got kicked out and wasn't able to get a final tally. This would almost certainly be one you would have to do one subfolder at a time.

**Comment by erikfriesen** [comment]
I ended up having to restart just before i went to be (screwup on my part). It's been runnng about 9 hours and is up to about 125k files in 49k directories.

**Comment by erikfriesen** [comment]
well, it was still running this afternoon. i cancelled the job and will move on to an easier task for now

**Comment by gofrogs2013** [comment]
I'm going one subfolder at a time now. Once I get it done I'll compress the entire thing and upload it.

**Comment by bkirkbri** [comment]
@gofrogs2013 Good idea. Please update here when complete (even if you only partially mirror it). Thanks!

**Comment by gofrogs2013** [comment]
@bkirkbri Sure thing. I've got one folder so far, the first of two folders within 1992, which alone is over 30K files! When done, I'll zip it and post a link.

**Comment by gofrogs2013** [comment]
Just finished the long download. I have the files and checksums here: https://github.com/climate-mirror/datasets/issues/279
